.wordlist.txt

```diff
@@ -29,12 +29,14 @@ balenaEtcher
 Bazarr
 Beelink
 Bitcoin
+Bittorrent
```

Comment:
Let's make a capital `T` in `BitTorrent`: https://www.bittorrent.com/

Since `BitTorrent` is used already, it seems to be in the aspell dictionary and hence the above wordlist entry can be removed.

Reply:
Ok.